#!/usr/bin/env python


PACKAGE = 'sss_data_generator'

import roslib
roslib.load_manifest(PACKAGE)

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('Amplitude', double_t, 0, "Amplitude parameter.", 430.0, 0.0, 1000.0)
gen.add('RandomAdd', double_t, 0, "Interval board for random.", 85.0, 0.0, 1000.0)
gen.add('Frequency', double_t, 0, "Frequency parameter.", 590.0, 0.0, 1000.0)
gen.add('MessageLength', int_t, 0, "The Length of sent message", 150, 1, 1000)

exit(gen.generate(PACKAGE, "sss_data_generator", "Params"))


